### ISSUE TITLE
Integration tests on stage are failing because `software_environment` table is not found

### ISSUE THREAD

**Issue by mayaCostantini** [thoth-station/storages#2628]
## Bug description
Integration tests on `ocp4-stage` are failing with the following error log:

```
Assertion Failed: Bad status code (500) when obtaining Thoth container images from https://stage.thoth-station.ninja/api/v1/container-images: {"details":{"datetime":"2022-04-22T00:10:41.298042","exception":{"args":[],"cause":null,"context":null,"traceback":["Traceback (most recent call last):\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/engine/base.py\", line 1276, in _execute_context\n    self.dialect.do_execute(\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/engine/default.py\", line 608, in do_execute\n    cursor.execute(statement, parameters)\n","psycopg2.errors.UndefinedTable: relation \"software_environment\" does not exist\nLINE 3: FROM software_environment) AS anon_1\n             ^\n\n","\nThe above exception was the direct cause of the following exception:\n\n","Traceback (most recent call last):\n","  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 2077, in wsgi_app\n    response = self.full_dispatch_request()\n","  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1525, in full_dispatch_request\n    rv = self.handle_user_exception(e)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/flask_cors/extension.py\", line 165, in wrapped_function\n    return cors_after_request(app.make_response(f(*args, **kwargs)))\n","  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1523, in full_dispatch_request\n    rv = self.dispatch_request()\n","  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1509, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/decorator.py\", line 68, in wrapper\n    response = function(request)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/uri_parsing.py\", line 149, in wrapper\n    response = function(request)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/validation.py\", line 399, in wrapper\n    return function(request)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/response.py\", line 112, in wrapper\n    response = function(request)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/parameter.py\", line 116, in wrapper\n    return function(**kwargs)\n","  File \"/opt/app-root/src/thoth/user_api/api_v1.py\", line 243, in list_thoth_container_images\n    entries_count = GRAPH.get_software_environments_count_all(\n","  File \"/opt/app-root/lib64/python3.8/site-packages/thoth/storages/graph/postgres.py\", line 2910, in get_software_environments_count_all\n    return query.count()\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/orm/query.py\", line 3803, in count\n    return self.from_self(col).scalar()\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/orm/query.py\", line 3523, in scalar\n    ret = self.one()\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/orm/query.py\", line 3490, in one\n    ret = self.one_or_none()\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/orm/query.py\", line 3459, in one_or_none\n    ret = list(self)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/orm/query.py\", line 3535, in __iter__\n    return self._execute_and_instances(context)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/orm/query.py\", line 3560, in _execute_and_instances\n    result = conn.execute(querycontext.statement, self._params)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/engine/base.py\", line 1011, in execute\n    return meth(self, multiparams, params)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/sql/elements.py\", line 298, in _execute_on_connection\n    return connection._execute_clauseelement(self, multiparams, params)\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/engine/base.py\", line 1124, in _execute_clauseelement\n    ret = self._execute_context(\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/engine/base.py\", line 1316, in _execute_context\n    self._handle_dbapi_exception(\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/engine/base.py\", line 1510, in _handle_dbapi_exception\n    util.raise_(\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/util/compat.py\", line 182, in raise_\n    raise exception\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/engine/base.py\", line 1276, in _execute_context\n    self.dialect.do_execute(\n","  File \"/opt/app-root/lib64/python3.8/site-packages/sqlalchemy/engine/default.py\", line 608, in do_execute\n    cursor.execute(statement, parameters)\n","sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation \"software_environment\" does not exist\nLINE 3: FROM software_environment) AS anon_1\n             ^\n\n[SQL: SELECT count(*) AS count_1 \nFROM (SELECT software_environment.id AS software_environment_id, software_environment.environment_name AS software_environment_environment_name, software_environment.python_version AS software_environment_python_version, software_environment.image_name AS software_environment_image_name, software_environment.image_sha AS software_environment_image_sha, software_environment.os_name AS software_environment_os_name, software_environment.os_version AS software_environment_os_version, software_environment.thoth_image_name AS software_environment_thoth_image_name, software_environment.thoth_image_version AS software_environment_thoth_image_version, software_environment.env_image_name AS software_environment_env_image_name, software_environment.env_image_tag AS software_environment_env_image_tag, software_environment.cuda_version AS software_environment_cuda_version, software_environment.environment_type AS software_environment_environment_type \nFROM software_environment) AS anon_1]\n(Background on this error at: http://sqlalche.me/e/13/f405)\n"]},"type":"InternalServerError"},"error":"Internal server error occurred"}
```

It looks like the `software_environment` table was not found when trying to list the available Thoth container images.

### Steps to Reproduce
See the last integration test results (2022-04-22 version 0.11.2) for `ocp4-stage` for the Thoth container images feature.

**Comment by harshad16** [comment]
![postgresl](https://user-images.githubusercontent.com/14028058/164628505-03749fdd-85ad-4555-a988-2161110afb51.png)
The PostgreSQL table exists in stage, seems like storages is having trouble accessing it.

**Comment by mayaCostantini** [comment]
@harshad16 I tested getting available container images with the [`get_software_environments_all`](get_software_environments_all) method locally as done when the request is posted through the user API and it seemed to work properly. I also updated the database schema to its latest version before doing so.
Getting images from the `/container-images` endpoint on the stage API worked as well.
I will continue investigating on the issue, thanks for looking into it :+1:
